### PR TITLE
docs: update for connected and disconnected

### PR DESCRIPTION
**Pull request by tenthirtyam**
### Summary

Updates the documentation to provide instructions for:
- Connected Environment (internet access)
- Disconnected Environment (dark-site / air-gapped)

### Type

- [ ] Bugfix
- [ ] Enhancement or Feature
- [ ] Code Style or Formatting
- [x] Documentation
- [ ] Refactoring
- [ ] Chore
- [ ] Other
        Please describe:

### Breaking Changes?

- [ ] Yes, there are breaking changes.
- [x] No, there are no breaking changes.

### Test and Documentation

image

- [x] Tests have been completed.
- [x] Documentation has been added or updated.

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -1,82 +1,83 @@
...
 
-    ```powershell
-    --8<-- "./docs/snippets/save-module-local-windows.ps1"
-    ```
+        ```powershell
+        --8<-- "./docs/snippets/save-module-local-windows.ps1"
+        ```
 
-    Copy the PowerShell Modules [module dependencies](/powershell-module-for-vmware-cloud-foundation-password-management/#module-dependencies) from the Local Machine to air-gapped facing machine by running the following commands:
+        From the system with the Internet connection, copy the module dependencies to a target system by running the following commands in the PowerShell console:
```
Comment (bhumitra) [inline docs/install.md:42]:
```suggestion
        From a system with an Internet connection, copy the module dependencies to a target system by running the following commands in the PowerShell console:
```

File: docs/install.md [hunk 2]
```diff
@@ -1,82 +1,83 @@
...
 
-    ```powershell
-    --8<-- "./docs/snippets/save-module-local-linux.ps1"
-    ```
+        ```powershell
+        --8<-- "./docs/snippets/save-module-local-linux.ps1"
+        ```
 
-    Copy the PowerShell Modules [module dependencies](/powershell-module-for-vmware-cloud-foundation-password-management/#module-dependencies) from the Local Machine to air-gapped facing machine by running the following commands:
+        From the system with an Internet connection, copy the module dependencies to a target system by running the following commands in the PowerShell console:
```
Comment (bhumitra) [inline docs/install.md:68]:
```suggestion
        From a system with an Internet connection, copy the module dependencies to a target system by running the following commands in the PowerShell console:
```

**Review by bhumitra — CHANGES_REQUESTED** [review]
Just a couple of minor changes. Rest is all good to go.

File: docs/install.md (hunk 1 above)
Comment (tenthirtyam) [reply docs/install.md:42]: This was intententially worded this way as a follow up from the first step and it being the same system. :)

File: docs/install.md (hunk 2 above)
Comment (tenthirtyam) [reply docs/install.md:68]: This was intententially worded this way as a follow up from the first step and it being the same system. :)

**Pull request merged**